pages/[network]/connect-account.tsx

```diff
@@ -43,7 +43,7 @@ export default function ConnectAccount() {
   const { t } = useTranslation(["common", "connect-account"]);
 
   const [isGhValid, setIsGhValid] = useState(null);
-  const [lastAddressBeforeConnect, setLastAddressBeforeConnect] = useState("");
+  //const [lastAddressBeforeConnect, setLastAddressBeforeConnect] = useState("");
```

Comment:
?

Reply:
is not being used. but I will improve the text and add the disable lint